plugins/package-managers/node/src/main/kotlin/Pnpm.kt

```diff
@@ -70,7 +70,7 @@ class Pnpm(
 
     override fun command(workingDir: File?) = if (Os.isWindows) "pnpm.cmd" else "pnpm"
 
-    override fun getVersionRequirement(): RangesList = RangesListFactory.create("5.* - 7.*")
+    override fun getVersionRequirement(): RangesList = RangesListFactory.create("5.* - 8.*")
```

Comment:
The change of the default resolution mode in the change log IIUC may lead to change in dependency resolution results. However, I believe if the lockfile practice is followed, it shouldn't have any effect.